### PR TITLE
Kill kGraft job if patch isn't correctly applied

### PR DESCRIPTION
**Pull request by mimi1vx**

### CODE REVIEW

File: tests/kernel/boot_ltp.pm [hunk 1]
```diff
@@ -37,6 +37,11 @@ sub run {
         assert_script_run("lsblk -la; export LTP_BIG_DEV=$block_dev");
     }
 
+    # check kGraft patch if KGRAFT=1
+    if (check_var('KGRAFT', '1')) {
```
Comment (rwx788) [inline tests/kernel/boot_ltp.pm]: As '1' can be treated as true, maybe we can use get_var and then we can transform it to one liner. But it's matter of taste.

**Comment by rwx788** [comment]
Please rebase and add description to PR and commit. Verification run also won't harm. Thanks!

**Comment by mimi1vx** [comment]
@rwx788 rebase ?
why ? is rebased to master few hours ago. And commit messages are clear

**Comment by rwx788** [comment]
@mimi1vx I'm not sure why you want to have 2 commits for this PR. Commit messages themselves are good, but it's always good to have a small description which provides more details what was aimed etc. Anyway, fine with me.

**Comment by mimi1vx** [comment]
@rwx788 ah, you mean `squash` commits :) from my point view are two different changes.

First was check if is kgraft patch correctly loaded, we found a case where was installed but not correctly loaded after boot.

And the second commit is because Live Patching from SLE12SP3 is extension, not module

**Comment by rwx788** [comment]
@mimi1vx oh, yes, sorry for the confusion ;)

File: tests/kernel/boot_ltp.pm (hunk 1 above)
Comment (okurz) [inline tests/kernel/boot_ltp.pm]: Yeah, you should really use `get_var()` or is there a specific reason you check for equality?

**Pull request merged**